src/main/java/bio/terra/service/snapshot/flight/export/cleanUpExportGsPathsStep.java

```diff
@@ -0,0 +1,51 @@
+package bio.terra.service.snapshot.flight.export;
+
+import bio.terra.service.filedata.google.gcs.GcsPdao;
+import bio.terra.service.resourcemanagement.google.GoogleBucketResource;
+import bio.terra.service.snapshot.Snapshot;
+import bio.terra.service.snapshot.SnapshotService;
+import bio.terra.service.snapshot.flight.SnapshotWorkingMapKeys;
+import bio.terra.service.tabulardata.google.BigQueryPdao;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import bio.terra.stairway.exception.RetryException;
+import java.util.UUID;
+
+public class cleanUpExportGsPathsStep implements Step {
+
+  private final BigQueryPdao bigQueryPdao;
+  private final GcsPdao gcsPdao;
+  private final SnapshotService snapshotService;
+  private final UUID snapshotId;
+
+  public cleanUpExportGsPathsStep(
+      BigQueryPdao bigQueryPdao,
+      GcsPdao gcsPdao,
+      SnapshotService snapshotService,
+      UUID snapshotId) {
+
+    this.bigQueryPdao = bigQueryPdao;
+    this.gcsPdao = gcsPdao;
+    this.snapshotService = snapshotService;
+    this.snapshotId = snapshotId;
+  }
+
+  @Override
+  public StepResult doStep(FlightContext context) throws InterruptedException, RetryException {
+    Snapshot snapshot = snapshotService.retrieve(snapshotId);
+    bigQueryPdao.deleteFirestoreGsPathExternalTable(snapshot, context.getFlightId());
+
+    GoogleBucketResource exportBucket =
+        context
+            .getWorkingMap()
+            .get(SnapshotWorkingMapKeys.SNAPSHOT_EXPORT_BUCKET, GoogleBucketResource.class);
+    gcsPdao.deleteFileByName(exportBucket, SnapshotExportDumpFirestoreStep.getFileName(context));
```

Comment:
It's strange to me to call another step's method. Could getFileName() be moved to the service layer SnapshotService?) or a util?